deployments/hadoop-yarn/ansible/27-install-zeppelin.yml

```
@@ -274,6 +278,10 @@
         group: "{{zepuser}}"
         mode: 0775
 
-    - name: "Add group for Zeppelin Users"
+    - name: Set folder permissions to 0775
       become: true
-      command: groupadd {{ zepusersgroup }}
+      file: path={{ item }} mode=0750 state=directory group={{zepusersgroup}}
```

Human readable text is better than numeric.
Useful [calculator tool](https://ss64.com/bash/chmod.html)

```suggestion
      file: path={{ item }} mode='u=rwx,g=rx,o=' state=directory group={{zepusersgroup}}
```